File: 26-ckql/python-sdk/ckql.py
import os, requests
API = "https://api.costkatana.com/api"
KEY = os.getenv("COST_KATANA_API_KEY")

def execute_query(query):
    res = requests.post(f"{API}/ckql/query",
        json={"query": query},
        headers={"Authorization": f"Bearer {KEY}"})
    res.raise_for_status()
    print(f"✅ Query executed: {len(res.json()['data'])} rows")
    return res.json()["data"]

def get_schema():
    res = requests.get(f"{API}/ckql/schema",
        headers={"Authorization": f"Bearer {KEY}"})
    res.raise_for_status()
    print(f"📊 Schema: {', '.join(res.json()['data'].keys())}")
    return res.json()["data"]

File: 26-ckql/npm-package/ckql.ts
import axios from 'axios';
const API = 'https://api.costkatana.com/api'\;
const KEY = process.env.COST_KATANA_API_KEY;

export async function executeQuery(query: string) {
  const res = await axios.post(`${API}/ckql/query`, {query},
    {headers: {'Authorization': `Bearer ${KEY}`}});
  console.log('✅ Query executed:', res.data.data.length, 'rows');
  return res.data.data;
}

export async function getSchema() {
  const res = await axios.get(`${API}/ckql/schema`,
    {headers: {'Authorization': `Bearer ${KEY}`}});
  console.log('📊 Schema:', Object.keys(res.data.data).join(', '));
  return res.data.data;
}
